providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -185,16 +187,54 @@ static int dh_exportkey(void *key, OSSL_CALLBACK *param_cb, void *cbarg)
     return ret;
 }
 
+/*
+ * Same function for domain parameters and for keys.
+ * "dpk" = "domain parameters & keys"
+ */
+static int dh_get_dpk_params(void *key, OSSL_PARAM params[])
```

Comment:
Nit: should we add `ossl_inline`? (and do the same for the other keymgmt)

Reply:
Sure